fastlane/Fastfile

```diff
@@ -798,16 +801,15 @@ end
 ########################################################################
 # Fastlane match code signing
 ########################################################################
+
+  APP_STORE_BUNDLE_IDENTIFIER = 'com.codality.NotationalFlow'.freeze
```

Comment:
Seems strange to have this constant declared here in the middle of the file while it's used _before_.

I can totally see the appeal of making it close to the "Fastlane match code signing" section, but since it's also used in lanes that are declared earlier in the file, I think this should be moved at the top of the Fastfile instead.

Reply:
Good point 👍